plugins/media.js

```diff
@@ -0,0 +1,20 @@
+import { oEmbeddable } from './oembed.js';
+
+export function isPDF(mimeType) {
+  return mimeType === 'application/pdf';
+}
+
+export function isHTMLVideo(mimeType, codec) {
+  return (mimeType === 'video/ogg') ||
+    (mimeType === 'video/webm') ||
+    ((mimeType === 'video/mp4') && (codec === 'h264'));
+}
+
+export function isOEmbed(oembedUrl) {
+  return oEmbeddable(oembedUrl);
+}
+
+// TODO: as the pdf is currently just an image with a link, it is not marked as "rich media", this might change in the future
+export function isRichMedia(mimeType, codec, oembedUrl) {
```

Comment:
Let's change the param name from `oembedUrl` to `mediaUrl` as it may or may not be oEmbeddable.